index.js

```diff
@@ -62,10 +63,10 @@ Merry.prototype.start = function () {
 }
 
 Merry.prototype.listen = function (port) {
-  assert.equal(typeof port, 'number', 'Merry.listen: port should be type number')
+  this._port = port || process.env.PORT || this.env.PORT
+  assert.ok(this._port, 'Merry.listen: port should exist')
```

Comment:
ok, this is fine but I am still thinking we should enforce that PORT should be a number otherwise `server.listen` won't work. but really this is up to you 😅  I'm fine either way; me personally I would use something like:

`Number(process.env.PORT)`

then run the check that you had before 😉